streams-contrib/streams-persist-elasticsearch/src/main/java/org/apache/streams/elasticsearch/ElasticsearchMetadataUtil.java

```diff
@@ -100,6 +100,18 @@ public static String getId(StreamsDatum datum) {
         return id;
     }
 
+    public static String getParent(StreamsDatum datum) {
+
+        String parent = null;
+
+        Map<String, Object> metadata = datum.getMetadata();
+
+        if( parent == null && metadata != null && metadata.containsKey("parent"))
+            parent = (String) datum.getMetadata().get("parent");
```

Comment:
Small nit: Why not just pull this out of "metadata" instead of using "datum.getMetadata()"?